bindings/rust/wasmedge-sys/src/value.rs

```diff
@@ -8,6 +10,30 @@ pub enum Value {
     I64(i64),
     F32(f32),
     F64(f64),
+    V128(u128),
+    /// A reference to a Wasm function.
+    FuncRef(u128),
+    /// A reference to opaque data in the Wasm instance.
+    ExternRef(u128),
```

Comment:
I see that go-sdk is relatively complex in adding the implementation of the externref part, and also adding support for linear memory management for webassembly. This is another thing I was confused about while working on this feature, doesn't the Rust side need to be consistent with go-sdk?

Reply:
I thought the host function depended on the memory management capabilities of externref, but now it looks like it doesn't.